src/helpers/getPaginationLink.js

```diff
@@ -1,3 +1,11 @@
-export default (page, qs) => {
-  return `/petitions/page/${page}${qs ? `?${qs}` : ''}`;
+export default (params, page, qs) => {
+  var petitionsPath = '/petitions/';
+
+  if (params.cityName && params.city) {
+    petitionsPath += `${params.cityName}-${params.city}/`;
+  }
+
+  petitionsPath += `page/${page}${qs ? `?${qs}` : ''}`;
```

Comment:
sorry... I added a bullshit comment not understanding the code, and then deleted it!